src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -133,6 +134,12 @@ function ReportActionItemMessageEdit(
     const emojiPickerSelectionRef = useRef<Selection | undefined>(undefined);
     // The ref to check whether the comment saving is in progress
     const isCommentPendingSaved = useRef(false);
+    const originalReportID = getOriginalReportID(reportID, action);
+    const [allReports] = useOnyx(ONYXKEYS.COLLECTION.REPORT, {canBeMissing: true});
+    const originalReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${originalReportID}`];
```

Comment:
Rather than subscribe to the entire collect, can you connect to the key for the original report ID and use the `DEFAULT_NUMBER_ID` as a fallback?